toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -802,7 +813,12 @@ func (b *LiveOSIsoBuilder) prepareLiveOSDir(inputSavedConfigsFilePath string, wr
 		return err
 	}
 
-	b.artifacts.dracutPackageInfo, err = getDracutVersion(writeableRootfsDir)
+	b.artifacts.dracutPackageInfo, err = getPackageInformationFromRootfsDir(writeableRootfsDir, "dracut")
```

Comment:
`getPackageInformationFromRootfsDir` sets up the chroot environment. Doing that twice in a row seems wasteful.

Reply:
pending

Reply:
fixed.